lib/ini.js

```diff
@@ -1,4 +1,5 @@
 const { hasOwnProperty } = Object.prototype
+const commentsDictionary = {}
```

Comment:
I don't think this will work to have as a top level dictionary shared across invocations.

If i'm reading it right, if two documents are parsed and they have  keys that collide, the comments could end up in the wrong document when stringified in a different order.